Doc/using/mac.rst

```diff
@@ -114,15 +114,20 @@ Close this terminal window and the installer window.
 
 A default install will include:
 
-* A |usemac_applications_folder_name| folder in your :file:`Applications` folder. In here
-  you find :program:`IDLE`, the development environment that is a standard part of official
-  Python distributions; and :program:`Python Launcher`, which handles double-clicking Python
-  scripts from the macOS `Finder <https://support.apple.com/en-us/HT201732>`_.
-
-* A framework :file:`/Library/Frameworks/Python.framework`, which includes the
-  Python executable and libraries. The installer adds this location to your shell
-  path. To uninstall Python, you can remove these three things.
-  Symlinks to the Python executable are placed in :file:`/usr/local/bin/`.
+* A |usemac_applications_folder_name| folder in your :file:`Applications` folder;
+
+* Within the |usemac_applications_folder_name| folder: :program:`IDLE`, the development environment that is a standard part of official Python distributions;
+
+* Also within the |usemac_applications_folder_name| folder: :program:`Python Launcher`, which handles double-clicking
+  Python scripts from the macOS `Finder <https://support.apple.com/en-us/HT201732>`_;
+
+* A :file:`3.13` folder (under the "framework" :file:`/Library/Frameworks/Python.framework`), which includes the
```

Comment:
Rather than hardcoding `3.13` which is only correct for a single branch, let's use a "replacement":

```suggestion
* A |usemac_x_dot_y_literal| folder (under the "framework" :file:`/Library/Frameworks/Python.framework`), which includes the
```

And add it with the others in this file:

```diff
 .. |usemac_x_dot_y| replace:: 3.13
+.. |usemac_x_dot_y_literal| replace:: ``3.13``
 .. |usemac_python_x_dot_y_literal| replace:: ``python3.13``
 .. |usemac_python_x_dot_y_t_literal| replace:: ``python3.13t``
 .. |usemac_python_x_dot_y_t_literal_config| replace:: ``python3.13t-config``
```

And actually, those are still hardcoded, so let's remove the `usemac_` prefix and integrate them into the `rst_epilog` which uses the _current_ version number:

https://github.com/python/cpython/blob/be8ae086874cac42eea104c1ff21ef5868d50bdd/Doc/conf.py#L77-L80

The `rst_epilog` replacements are available for all pages.

Reply:
This is surely The Right Thing®, but I'm unfamiliar enough to ReStructured Text to do it myself (I took a stab at it and found myself going in circles after an hour).

Reply:
That's fine, we can do it in a followup. Thank you for trying!